test/Microsoft.AspNetCore.Authentication.Core.Test/TokenExtensionTests.cs

```diff
@@ -117,7 +122,68 @@ public void UpdateTokenValueReturnsFalseForUnknownToken()
             Assert.Null(props.GetTokenValue("ONE"));
             Assert.Null(props.GetTokenValue("Jigglypuff"));
```

Comment:
Oh I recognized it. Testing on Pokemon is ethically discouraged, that's what Chell is for.